test/plugins/test_lyrics.py

```diff
@@ -22,7 +22,12 @@
 
 from beets.library import Item
 from beets.test.helper import PluginMixin
-from beetsplug import lyrics
+
+try:
+    from beetsplug import lyrics
```

Comment:
Lyrics integration tests only run if there have been adjustments in one of the lyrics source files.

Did you add this because these tests have been failing locally?

Reply:
Indeed, langdetect is not installed by poe. Atleast it wasn't for me. Therefore the tests fail.

I thought it is reasonable to just skip it, if the optional dependency is not installed.

I can revert the change if you want, dont mind either way.

Reply:
I see what you mean. There's a small issue here - we want the opposite in CI: if any dependencies are missing it should fail, forcing us to fix our CI setup and installing the missing dependency.

See `test_autobpm.py` for an example how this could be solved.